### PR TITLE
Bybit inverse futures

### PR DESCRIPTION
**Pull request by Stolenko**

### CODE REVIEW

File: js/bybit.js [hunk 1]
```diff
@@ -1218,15 +1223,28 @@ module.exports = class bybit extends Exchange {
...
         const marketType = this.safeString (marketTypes, symbol);
-        let method = (marketType === 'linear') ? 'privateLinearGetOrderSearch' : 'v2PrivateGetOrder';
+        let method = undefined;
+        if (marketType === 'linear') {
+            method = 'privateLinearGetOrderSearch';
+        } else if (marketType === 'inverse') {
+            method = 'v2PrivateGetOrder';
+        } else if (marketType === 'inverseFuture') {
+            method = 'futuresPrivateGetOrder';
+        }
```
Comment (mrbaguvix) [inline js/bybit.js:1233]: Avoid the use of else if statement here.
```suggestion
        if (marketType === 'linear') {
            method = 'privateLinearGetOrderSearch';
        }
        if (marketType === 'inverse') {
            method = 'v2PrivateGetOrder';
        }
        if (marketType === 'inverseFuture') {
            method = 'futuresPrivateGetOrder';
        }
```

**Review by mrbaguvix — CHANGES_REQUESTED** [review]
Avoid the use of `else if` statement in your code. They don't make things flexible.

**Comment by kroitor** [comment]
@Stolenko thx, merging shortly!

File: js/bybit.js (hunk 1 above)
Comment (Stolenko) [reply js/bybit.js]: fixed

Comment (Stolenko) [reply js/bybit.js:1237]: @kroitor done

**Pull request merged**